Team Split Test Bug

Looks like the compiler is right about this. Should we add error checks or initialize the dims to 1?
```
../../../test/spec-example/shmem_team_split_2D.c: In function ‘main’:
../../../test/spec-example/shmem_team_split_2D.c:56:3: warning: ‘zdim’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   56 |   for (int zdx = 0; zdx < zdim; zdx++) {
      |   ^~~
../../../test/spec-example/shmem_team_split_2D.c:57:5: warning: ‘ydim’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   57 |     for (int ydx = 0; ydx < ydim; ydx++) {
      |     ^~~
../../../test/spec-example/shmem_team_split_2D.c:58:7: warning: ‘xdim’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   58 |       for (int xdx = 0; xdx < xdim; xdx++) {
      |       ^~~
```

--- a/test/spec-example/shmem_team_split_2D.c
+++ b/test/spec-example/shmem_team_split_2D.c
@@ -20,6 +20,7 @@ static void find_xy_dims(int npes, int *x, int *y) {
 /*  Find x, y, and z such that x * y * z == npes and
  *  abs(x - y) + abs(x - z) + abs(y - z) is minimized.  */
 static void find_xyz_dims(int npes, int *x, int *y, int *z) {
+  *x = *y = *z = 1;
   for(int divider = ceil(cbrt(npes)); divider >= 1; divider--)
     if (npes % divider == 0) {
       *x = divider;
@@ -29,9 +30,7 @@ static void find_xyz_dims(int npes, int *x, int *y, int *z) {
 }
 
 int main(void) {
-  int xdim = 1;
-  int ydim = 1;
-  int zdim = 1;
+  int xdim, ydim, zdim;
 
   shmem_init();
   int mype = shmem_my_pe();

